modules/gridmap/editor/grid_map_editor_plugin.cpp

```diff
@@ -743,6 +743,16 @@ EditorPlugin::AfterGUIInput GridMapEditor::forward_spatial_input_event(Camera3D
 				}
 			}
 
+			// Swallow input to avoid conflicts with other plugins
+			for (int itemIdx = 0; itemIdx < options->get_popup()->get_item_count(); ++itemIdx) {
```

Comment:
```suggestion
			for (int item_idx = 0; item_idx < options->get_popup()->get_item_count(); ++item_idx) {
```
Use `snake_case` only for variable names.
(it could be simply `i`)